src/Core/src/TaskExtensions.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Maui
 {
 	public static class TaskExtensions
 	{
-		public static async void FireAndForget(
+		internal static async void FireAndForget(
```

Comment:
Since we started using IVT we'll just make this internal. No reason to expose these to users